js/src/carousel.js

```diff
@@ -161,7 +161,7 @@ const Carousel = (($) => {
 
       if (this._config.interval && !this._isPaused) {
         this._interval = setInterval(
-          $.proxy(document.visibilityState ? this.nextWhenVisible : this.next, this), this._config.interval
+          (document.visibilityState ? this.nextWhenVisible : this.next).bind(this), this._config.interval
```

Comment:
The other way to do this is `(document.visibilityState ? () => this.nextWhenVisible() : () => this.next())`, but that seemed cringeworthy.